src/gripper_controller/gripper_controller.py

```diff
@@ -77,7 +77,7 @@ def _open_gripper(self):
         return out
 
     def _close_gripper(self):
-        effort = 0.1  # 0.1 Newton is enough for nearly everything
+        effort = 0.15  # 0.1 Newton is enough for nearly everything
```

Comment:
Update the comment as well.
Question: we want to add a variable gripper force, correct? So will this be overwritten by a later PR?

Reply:
Yep it will be.